Samples/DotNetCore/XenaSamples.HybridFlow.MvcClient/Startup.cs

```diff
@@ -33,24 +34,25 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
-
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
+            services.AddXenaAPI(new XenaApiOptions() { XenaEndpoint = "https://test.xena.biz/api"});
```

Comment:
De her samples skal være offentlige? Er vi sikre på at xena test skal fremgå her?

Reply:
... ok if you insist.